sdk/networkanalytics/arm-networkanalytics/review/arm-networkanalytics.api.md

```diff
@@ -149,156 +150,102 @@ export interface DataProductsCatalogProperties {
+    listBySubscription: (subscriptionId: string, options?: DataProductsCatalogsListBySubscriptionOptions) => PagedAsyncIterableIterator<DataProductsCatalog>;
 }
 
-// @public
-export type DataProductsCreateResponse = DataProduct;
-
-// @public
-export interface DataProductsDeleteHeaders {
-    location?: string;
-    retryAfter?: number;
+// @public (undocumented)
+export interface DataProductsCreateOptions extends OperationOptions {
 }
 
-// @public
-export interface DataProductsDeleteOptionalParams extends coreClient.OperationOptions {
-    resumeFrom?: string;
+// @public (undocumented)
+export interface DataProductsDeleteOperationOptions extends OperationOptions {
     updateIntervalInMs?: number;
 }
 
-// @public
-export type DataProductsDeleteResponse = DataProductsDeleteHeaders;
-
-// @public
-export interface DataProductsGenerateStorageAccountSasTokenOptionalParams extends coreClient.OperationOptions {
-}
-
-// @public
-export type DataProductsGenerateStorageAccountSasTokenResponse = AccountSasToken;
-
-// @public
-export interface DataProductsGetOptionalParams extends coreClient.OperationOptions {
-}
-
-// @public
-export type DataProductsGetResponse = DataProduct;
-
-// @public
-export interface DataProductsListByResourceGroupNextOptionalParams extends coreClient.OperationOptions {
+// @public (undocumented)
+export interface DataProductsGenerateStorageAccountSasTokenOptions extends OperationOptions {
 }
 
-// @public
-export type DataProductsListByResourceGroupNextResponse = DataProductListResult;
-
-// @public
-export interface DataProductsListByResourceGroupOptionalParams extends coreClient.OperationOptions {
+// @public (undocumented)
+export interface DataProductsGetOptions extends OperationOptions {
 }
 
-// @public
-export type DataProductsListByResourceGroupResponse = DataProductListResult;
-
-// @public
-export interface DataProductsListBySubscriptionNextOptionalParams extends coreClient.OperationOptions {
+// @public (undocumented)
+export interface DataProductsListByResourceGroupOptions extends OperationOptions {
 }
 
-// @public
-export type DataProductsListBySubscriptionNextResponse = DataProductListResult;
-
-// @public
-export interface DataProductsListBySubscriptionOptionalParams extends coreClient.OperationOptions {
+// @public (undocumented)
+export interface DataProductsListBySubscriptionOptions extends OperationOptions {
 }
 
-// @public
-export type DataProductsListBySubscriptionResponse = DataProductListResult;
-
-// @public
-export interface DataProductsListRolesAssignmentsOptionalParams extends coreClient.OperationOptions {
+// @public (undocumented)
+export interface DataProductsListRolesAssignmentsOptions extends OperationOptions {
 }
 
-// @public
-export type DataProductsListRolesAssignmentsResponse = ListRoleAssignments;
-
-// @public
-export interface DataProductsRemoveUserRoleOptionalParams extends coreClient.OperationOptions {
+// @public (undocumented)
+export interface DataProductsOperations {
```

Comment:
could you change manually from DataProductsOperations to DataProducts? so we could better find the gaps.